src/less/core/global.less

```diff
@@ -21,15 +21,15 @@
 // Body reset
 
 html {
-    // font-size: 10px;
+    font-size: 16px;
```

Comment:
Setting the base font-size here overrides the users browsers font-size.
The default is 16px, so do you need to set it?